code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -436,13 +436,19 @@
 	pixel_x = -64
 	pixel_y = -64
 	unacidable = 1
-	var/mob/living/immune = null // the one who creates the timestop is immune
+	var/mob/living/immune = list() // the one who creates the timestop is immune
 	var/freezerange = 2
 	var/duration = 140
 	alpha = 125
 
 /obj/effect/timestop/New()
 	..()
+	for(var/mob/M in player_list)
```

Comment:
It was what I intended, but I guess it would be funny to see the xenobiologist counter timestop.

Not that they'll ever get far enough in the evolution tree during a wizard round.